app/views/helpers/calcBreakdownHelper.scala.html

```diff
@@ -112,17 +112,23 @@
         @if(calcModel.hasNISection) {
         <section id ="ni-section">
             @if(calcModel.hasNic2Amount) {
-                <tr @if(calcModel.hasNic4Amount){class="no-border-bottom"}>
+                <tr @if(calcModel.hasNic4Amount || calcModel.hasTaxReliefs){class="no-border-bottom"}>
                     <td id="nic2-amount-heading">@messages("estimated_tax_liability.calc-breakdown.nic2")</td>
                     <td id="nic2-amount" class="summary-data">@breakdown.nic.class2.toCurrency</td>
                 </tr>
             }
             @if(calcModel.hasNic4Amount) {
-                <tr>
+                <tr @if(calcModel.hasTaxReliefs){class="no-border-bottom"}>
                     <td id="nic4-amount-heading">@messages("estimated_tax_liability.calc-breakdown.nic4")</td>
                     <td id="nic4-amount" class="summary-data">@breakdown.nic.class4.toCurrency</td>
                 </tr>
             }
+            @if(calcModel.hasTaxReliefs) {
```

Comment:
Can this be moved outside of the National Insurance section. It can be in it's own section called `tax-reliefs`